tests/functional/test_autocommit.py

```diff
@@ -31,12 +31,33 @@
 {% endmacro %}
 """
 
+_MACROS__UDPATE_MY_SEED = """
+{% macro udpate_my_seed() %}
```

Comment:
This looks like it's intended to be `_MACROS__UPDATE_MY_SEED` and `update_my_seed`.